lib/components/base-components/NormalComponent/NormalComponent_doInitialSilkscreenOverlapAdjustment.ts

```diff
@@ -0,0 +1,353 @@
+import type { NormalComponent } from "./NormalComponent"
+
+interface SilkscreenElement {
+  bounds: Bounds
+  layer: string
+  type: "text" | "path" | "rect" | "circle"
+  pcb_component_id?: string
+  id: string
```

Comment:
`id` is a banned word. Also this type name is too similar to other type names, i think you can get a better name e.g. `SilkscreenElementWithBounds`